src/vs/workbench/test/browser/workbenchTestServices.ts

```diff
@@ -173,6 +173,7 @@ import { IEditorPaneService } from 'vs/workbench/services/editor/common/editorPa
 import { EditorPaneService } from 'vs/workbench/services/editor/browser/editorPaneService';
 import { IContextViewService } from 'vs/platform/contextview/browser/contextView';
 import { ContextViewService } from 'vs/platform/contextview/browser/contextViewService';
+import { CustomEditorLabelService, ICustomEditorLabelService } from '../../common/editor/editorLabels';
```

Comment:
Somehow you eneded up with relative imports? Can you check if you have configured something in settings? We only use absolute imports in VSCode.

Reply:
Very weird!